explorer/site-config/anvil/dev/fileTransformer.ts

```diff
@@ -0,0 +1,78 @@
+import { concatStrings } from "app/utils/string";
+import { AnvilFileResponse } from "app/models/responses";
+import * as C from "../../../app/components";
+
+export const filesToDatasetNameColumn = (
+  file: AnvilFileResponse
+): React.ComponentProps<typeof C.Links> => {
+  if (!file.files) {
+    return {
+      links: [],
+    };
+  }
+
+  return {
+    links: [
```

Comment:
My apologies Bruce, I made a typo in the corresponding ticket: a detail page is _not_ required for files. Can you update this to just return the dataset name?

Reply:
Sure!